Source/CutomePlayer/YoutubeVideoPlayer.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  YoutubeVideoPlayer.swift
+//  edX
+//
+//  Created by Andrey on 9/4/18.
+//  Copyright © 2018 edX. All rights reserved.
+//
+
+class YoutubeVideoPlayer: VideoPlayer{
+
+    let playerView: YTPlayerView
+    var videoId: String
+
+    override var currentTime: TimeInterval {
+        return Double(playerView.currentTime())
+    }
+    override init(environment : Environment) {
+        playerView = YTPlayerView()
+        videoId = String()
+        super.init(environment: environment)
+        playerView.delegate = self
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .black
```

Comment:
VideoPlayer has a `movieBackgroundView` that sets the background to black with an alpha of 0.5 - do we need to match that here? (Or make it not fileprivate and use it here?